docs/errors.rst

```diff
@@ -34,27 +34,37 @@ these error messages and what you can do when they occur.
 Where do these errors come from?
 --------------------------------
 
-This error is caused by an internal invariant being violated during TVM's
-execution. On a technical level, the message is generated by the
-``ICHECK`` macro, found in ``include/tvm/runtime/logging.h``.
+TVM is complex system, so errors may come from many places, and may inference
+other errors. Most of the errors comes from runtime enviroment. From technical
+point of view, the message is generated by the ``ICHECK`` macro,
+found in ``include/tvm/runtime/logging.h``.
 The ``ICHECK`` macro is used in many places in the TVM code to assert
 some condition is true during execution; any time the assertion fails, TVM
 will exit with the error message shown above.
 
 For more details about how errors are handled and generated by TVM, please
-see `error-handling-guide`.
+see :ref:`error-handling-guide` .
+
+What if I'm using RPC and I'm getting errors?
+---------------------------------------------
+
+First of all, you should enable verbosity and configure logging. If you are
+using android devices, you should also look into adb logcat.
+
+
 
 What should I do when I encounter such an error?
 ------------------------------------------------
 
-First of all, *don't panic*. Well, you can panic, but it won't help.
+First of all, *don't panic*. You and most of TVM users are experienced developers,
+so you know how to read source code, how to read stacktraces and how to debug code.
```

Comment:
This sentence could be controversial, maybe replace it with a more "beginner friendly" one?